Content.Server/Salvage/SalvageSystem.Runner.cs

```diff
@@ -272,8 +275,36 @@ private void UpdateRunner()
             if (elimination.Megafauna.Count == 0)
             {
                 comp.Completed = true;
+            }
+        }
+
+        // give rewards if it was completed
+        var rewardQuery = EntityQueryEnumerator<SalvageExpeditionComponent>();
+        while (rewardQuery.MoveNext(out var uid, out var comp))
+        {
+            if (comp.Completed)
+            {
                 Announce(uid, Loc.GetString("salvage-expedition-completed"));
+                GiveReward(uid, comp);
             }
         }
     }
+
```

Comment:
Isn't this going to spam the reward every tick?

Reply:
isnt it deleted after